clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -495,10 +495,15 @@ static Error getSYCLDeviceLibs(SmallVector<std::string, 16> &DeviceLibFiles,
   return Error::success();
 }
 
-// This routine is used to convert SPIR-V input files into LLVM IR files.
-// If input is not a SPIR-V file, then the original file is returned.
-// TODO: Add a check to identify SPIR-V files and exit early if the input is
-// not a SPIR-V file.
+/// This routine is used to convert SPIR-V input files into LLVM IR files.
+/// 'llvm-spirv -r' command is used for this purpose.
+/// If input is not a SPIR-V file, then the original file is returned.
+/// TODO: Add a check to identify SPIR-V files and exit early if the input is
+/// not a SPIR-V file.
+/// 'Filename' is the input file that could be a SPIR-V file.
+/// 'Args' encompasses all arguments required for linking and wrapping device
+/// code and will be parsed to generate options required to be passed into the
+/// llvm-spirv tool.
```

Comment:
If we want to apply proper Doxygen annotations, we should add `\p` before the parameter name (or `\param` in extended form). See https://llvm.org/docs/CodingStandards.html#doxygen-use-in-documentation-comments for code examples:
```suggestion
/// \param Filename is the input file that could be a SPIR-V file.
/// \param Args encompasses all arguments required for linking and wrapping device
/// code and will be parsed to generate options required to be passed into the
/// llvm-spirv tool.
```

Reply:
I am hoping to wrap up this PR and do a 'global' comments update (for all my clang-linker-wrapper changes) as a follow-up step. Hope that's agreeable.

Thanks

Reply:
It will be cleaner to fix annotations for the comments added in this PR to avoid dependency on future changes (i.e. make this change self-contained). I don't want this to block the merge, so I'll approve and leave the decision on how to proceed with this pull request to @asudarsa and the other reviewers.